bin/potion

```diff
@@ -232,6 +235,12 @@ class PotionCommandLine
         end
       end
     end
+
+    def cdq_included?
+      return false unless File.exist?("Gemfile.lock")
+      data = IO.readlines("Gemfile.lock")
+      data.any?{|l| l.match(/\s.cdq\s\(/) }
```

Comment:
I thought about that, but decided it was an edge case. The Gemfile.lock is a more consistently formatted place to find it so it made it easy to parse. For example, if CDQ is included through a wrapper (say `redpotion-cdq` eventually?) this will still find it.